Mathlib/NumberTheory/Padics/RingHoms.lean

```diff
@@ -125,8 +125,8 @@ theorem norm_sub_modPart (h : ‖(r : ℚ_[p])‖ ≤ 1) : ‖(⟨r, h⟩ - modP
       Int.cast_sub]
     apply Subtype.coe_injective
     simp only [coe_mul, Subtype.coe_mk, coe_natCast]
-    rw_mod_cast [@Rat.mul_den_eq_num r]
-    rfl
+    norm_cast
+    simp
```

Comment:
Are any simp's in this file noticably slower? (A "visual impression" is enough for me.)

Reply:
Not by eyeball, but feel free to bench if you think that's helpful.